source/_patterns/01-elements/input/_input.variables.scss

```diff
@@ -1,5 +1,15 @@
-$icons-path: "../../../icons/" !default;
-@use "@db-ui/base/build/scss/variables" as *;
-@use "@db-ui/base/build/scss/helpers/functions" as *;
-
 $input---color: $db-colors-secondary-enabled !default;
+
+// TODO rename
+@mixin input-label-translation() {
+	$transformScale: -1.25;
+
+	transition: opacity 0.08s, transform 0.15s;
+	opacity: var(--db-input-focus-opacity);
+
+	transform: translate(
+			$db-spacing-fixed-xs,
+			calc($db-sizing-md * $transformScale)
+		)
+		scale(var(--db-input-label-scale));
+}
```

Comment:
This seems to have been replaced by `label-focus-animation` within `source/_patterns/01-elements/input/input.scss`, hasn't it ?

Reply:
updated in https://github.com/db-ui/core/pull/198/commits/9291c0e751599052a5c87dc858f92b0e719c1eaa